internal/pkg/argocd/diff/diff.go

```diff
@@ -0,0 +1,260 @@
+// Copyright 2022 The Go Authors. All rights reserved.
```

Comment:
Could we please mention somewhere that we did hard-fork this package?
Just for future readers of this code.

Something like: https://github.com/golangci/gofmt/tree/master/gofmt#hard-fork-of-gofmt or even in the comments.